mono/mini/interp/interp.c

```diff
@@ -702,6 +702,7 @@ interp_throw (ThreadContext *context, MonoException *ex, InterpFrame *frame, gco
 
 	MonoContext ctx;
 	memset (&ctx, 0, sizeof (MonoContext));
+	MONO_CONTEXT_SET_SP (&ctx, &ext);
 
```

Comment:
Why is this needed ? ctx should be 0 for interpreter frames, we use MonoStackFrameInfo.frame_addr to compare frames now.

Reply:
I added it to make this check happy:
https://github.com/mono/mono/blob/2c54579df918cf756f38962ac4a106d8a71a14c1/mono/mini/debugger-agent.c#L3213-L3214

`filter_state.ctx` is setup by this:
https://github.com/mono/mono/blob/2c54579df918cf756f38962ac4a106d8a71a14c1/mono/mini/mini-exceptions.c#L1983

is there a better solution?